code/modules/mob/living/silicon/ai/say.dm

```diff
@@ -116,11 +116,11 @@ var/const/VOX_DELAY = 600
 	//writepanic("[__FILE__].[__LINE__] ([src.type])([usr ? usr.ckey : ""]) \\/mob/living/silicon/ai/verb/announcement_help()  called tick#: [world.time]")
 
 
-	var/dat = "Here is a list of words you can type into the 'Announcement' button to create sentences to vocally announce to everyone on the same level at you.<BR> \
+	var/dat = list("Here is a list of words you can type into the 'Announcement' button to create sentences to vocally announce to everyone on the same level at you.<BR> \

```

Comment:
Byond does some shit similar to what it does with appearances assigning them unique ids so if you are concatenating strings into longer and longer strings with many concatenations it churns. Using "[1][2][3][...]" concatenates them at once preventing that churn.